frontend/public/locales/zh/ingress.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Not configured": "未配置",
+  "Host": "主机",
+  "Path": "路径",
+  "Service": "Service",
+  "Service port": "服务端口",
+  "Ingress details": "入站详情",
+  "TLS certificate": "TLS 证书",
+  "Ingress rules": "入站规则",
+  "These rules are handled by a routing layer (Ingress Controller) which is updated as the rules are modified. The Ingress controller implementation defines how headers and other metadata are forwarded or manipulated": "这些规则由路由层（Ingress 控制器）处理，路由层会随规则的修改而更新。 Ingress 控制器定义了如何转发或处理标头和其他元数据",
+  "Name": "名称",
+  "Namespace": "命名空间",
+  "Labels": "标签",
+  "Hosts": "Hosts",
```

Comment:
"Hosts" could be translated into Chinese.

Reply:
This has been updated.